### PR TITLE
CATTY-464 Change icons of side menu

### PR DESCRIPTION
**Pull request by zwerg4**
Changed the icons of the side menu. Added new label for Minimize/Maximize. Label and image changes when clicked.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [X] Include the name of the Jira ticket in the PR’s title
- [X] Verify that the Jira ticket is in the status *Ready for Development*
- [X] Include a summary of the changes plus the relevant context
- [X] Choose the proper base branch (*develop*)
- [X] Confirm that the changes follow the project’s coding guidelines
- [X] Verify that the changes generate no compiler or linter warnings
- [X] Perform a self-review of the changes
- [X] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [X] Confirm that new and existing unit tests pass locally
- [X] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [X] Stick to the project’s git workflow (rebase and squash your commits)
- [X] Verify that your changes do not have any conflicts with the base branch
- [X] After the PR, verify that all CI checks have passed
- [X] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by Bortol15 — CHANGES_REQUESTED** [review]
Hey @zwerg4 , your changes are primarily looking good! Either way, I found a small bug, the label and Image change in an incorrect way if you *restart* the game when the screen is maximized. (minimize Label and Button are shown and the screen mode is already minimized) The problem is that the project get's restartet without the set screenMode (-> always in kCatrobatHeaderScreenModeStretch) and `restartAction`never updates the Button and Label accordingly. So I guess the easiest fix would be to call the `setupMenuButtons` and `setupLabels` method in the `restartAction` (I tested it, it at least works ;)) or, which would be better to create a update method for both. It propably would be cleaner to restart the stage in the chosen Screen Mode, but that cost-benefit ration would be way off I guss. Do not hesitate to send me a message on Slack if my explanation is unclear in any way or if you want to talk about the problem.
EDIT: and maybe it also would be cleaner to make the label and Image depended on the actual screenMode and not on the label text, but thats just a proposal ;)

**Review by Bortol15 — APPROVED** [review]

File: src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+        button.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -15).isActive = true
+        stagePresenterViewController.menuMaximizedMinimizedLabel = button
+
+        button = UIButton(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
+        self.addSubview(button)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.rightAnchor.constraint(equalTo: self.rightAnchor).isActive = true
+        button.leftAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
+        button.bottomAnchor.constraint(equalTo: stagePresenterViewController.menuMaximizedMinimizedLabel.topAnchor, constant: -5).isActive = true
+        stagePresenterViewController.menuAspectRatioButton = button
```
Comment (Bortol15) [inline src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift:274]: Is it possible to right one or a few different (if one would be to complicated) methods where you pass everything you need -> for example the type of button (i.e menuAspectRationButton), the constant (5, -15), the anchor (left, right) and different labels? I get if it's more work than benefit, but it seems that there could be a code reduction.

File: src/Catty/ViewController/Stage/StagePresenterViewController.m [hunk 2]
```diff
@@ -435,7 +459,19 @@ - (void)showHideAxisAction:(UIButton*)sender
...
+        [self setupButtonWithButton:self.menuAspectRatioButton
+                    ImageNameNormal:[UIImage imageNamed:@"stage_dialog_button_aspect_ratio_close"]
+                        andSelector:@selector(manageAspectRatioAction:)];
+     } else {
+         self.project.header.screenMode = kCatrobatHeaderScreenModeStretch;
+         [self.menuMaximizedMinimizedLabel setTitle:kLocalizedMaximize forState:UIControlStateNormal];
+         [self setupButtonWithButton:self.menuAspectRatioButton
+                     ImageNameNormal:[UIImage imageNamed:@"stage_dialog_button_aspect_ratio"]
+                         andSelector:@selector(manageAspectRatioAction:)];
+     }
```
Comment (Bortol15) [inline src/Catty/ViewController/Stage/StagePresenterViewController.m:474]: @zwerg4 I guess there is a little bug, `self.menuMaximizedMinimizedLabel setTitle` gets only called if you click on the aspectRationButton. So if you maximize the view and the restart the game, the text of the label does not get updated. You could for example check the `project.header.screenMode` also in the `viewWillAppear` method I'd say 🤔

**Review by Bortol15 — CHANGES_REQUESTED** [review]

File: src/Catty/ViewController/Stage/StagePresenterViewController.m (hunk 2 above)
Comment (m-herold) [reply src/Catty/ViewController/Stage/StagePresenterViewController.m:485]: Great code review, thanks!

File: src/Catty/Resources/Localization/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -766,12 +766,12 @@
...
 
-/* paint */
-"Green" = "Green";
-
 /* No comment provided by engineer. */
 "green" = "green";
 
+/* paint */
+"Green" = "Green";
+
```
Comment (Bortol15) [inline src/Catty/Resources/Localization/en.lproj/Localizable.strings:774]: It probably doesn't matter, but what are these color-string changes to the same value as deleted above for?

**Review by Bortol15 — APPROVED** [review]

**Pull request merged**